openjdk-testing/src/main/java/org/conscrypt/testing/TestClient.java

```diff
@@ -39,8 +37,8 @@ public TestClient(SSLSocket socket) {
     public void start() {
         try {
             socket.startHandshake();
-            input = new BufferedInputStream(socket.getInputStream());
-            output = new BufferedOutputStream(socket.getOutputStream());
+            input = socket.getInputStream();
```

Comment:
I think we need to be careful here.  We need to put some lower bound on the size of the buffer we're reading from the socket (e.g. 8KiB). Otherwise, socket reads will dominate the benchmarks for small message sizes (e.g. 64B).